CRUDRecipeEF.BL.DL/Data/DataSeed.cs

```diff
@@ -1,13 +1,26 @@
 ﻿using System.Collections.Generic;
 using CRUDRecipeEF.BL.DL.Entities;
+using Microsoft.Extensions.Logging;
 
 namespace CRUDRecipeEF.BL.DL.Data
 {
-    public static class DataSeed
+    public class DataSeed : IDataSeed
```

Comment:
Since you really wanted to changw this but i have only seen "teachers" do it using the static i went to ask a developer to make sure my point is actually right. It should be static because that means one less dependancy, also it lowers the amount of code that we have right now for example setting up a ctor with fields arent needed since we only use them inside a method and do it once.

Reply:
Wanted to be able to inject a logger but guess the static one can be used